assets/src/edit-story/components/form/media.js

```diff
@@ -183,87 +75,24 @@ const MediaInput = forwardRef(
       [onChange, openMediaPicker]
     );
 
-    const ref = useRef();
-    const [isHovering, setIsHovering] = useState(false);
-    const [isFocused, setIsFocused] = useState(false);
-    const resettableProps = {
-      tabIndex: 0,
-      'aria-label': ariaLabel,
-      onFocus: () => setIsFocused(true),
-      onBlur: (evt) => setIsFocused(ref.current.contains(evt.relatedTarget)),
-      onPointerEnter: () => setIsHovering(true),
-      onPointerLeave: () => setIsHovering(false),
-    };
-
-    const isMenuVisible = isHovering || isFocused;
-
     return (
-      <Container
-        ref={ref}
-        className={className}
-        disabled={disabled}
-        circle={circle}
-        size={size}
+      <Input
+        onMenuOption={onOption}
+        menuOptions={dropdownOptions}
+        openMediaPicker={openMediaPicker}
+        ref={forwardedRef}
         {...rest}
-        {...(canReset && resettableProps)}
-      >
-        {value && !isMultiple ? (
-          <Img src={value} circle={circle} alt={alt} />
-        ) : (
-          <DefaultImage size={size} />
-        )}
-        {loading && <LoadingDots />}
-        {canReset && isMenuVisible && (
-          <DropDownMenu options={dropdownOptions} onOption={onOption} />
-        )}
-        {!canReset && (
-          <EditBtn
-            ref={forwardedRef}
-            onClick={openMediaPicker}
-            circle={circle}
-            aria-label={ariaLabel}
-          >
-            <EditIcon />
-          </EditBtn>
-        )}
-      </Container>
+      />
     );
   }
 );
 
 MediaInput.propTypes = {
-  className: PropTypes.string,
-  label: PropTypes.string,
-  value: PropTypes.any,
+  buttonInsertText: PropTypes.string,
+  menuOptions: PropTypes.array,
   onChange: PropTypes.func.isRequired,
-  onBlur: PropTypes.func,
-  disabled: PropTypes.bool,
-  size: PropTypes.number,
-  circle: PropTypes.bool,
-  ariaLabel: PropTypes.string,
   type: PropTypes.string,
-  buttonInsertText: PropTypes.string,
   title: PropTypes.string,
-  alt: PropTypes.string,
-  imgProps: PropTypes.object,
-  loading: PropTypes.bool,
-  canReset: PropTypes.bool,
-};
```

Comment:
So many properties before... we really don't need them? 😲

Reply:
Some of these are still used, yes, but from the `...rest`.